Лабораторная работа №3

Feendiers opened the pull request.

Feendiers committed:
Add files via upload

mixeme commented on Лабораторная работа №3/Поиск индекса элемента в списке товаров.py:
```diff
@@ -0,0 +1,15 @@
+# TODO Напишите функцию для поиска индекса товара
+
+def index(list, product):
+    for i in range(len(list)):
```
Не в полной мере использован выданный материал.

mixeme commented on Лабораторная работа №3/Поиск общих участников.py:
```diff
@@ -0,0 +1,19 @@
+participants_second_group = "Петров|Сидоров|Смирнов"
+
+# TODO Провеьте работу функции с разделителем отличным от запятой
+def find_common_participants(str1, str2, sept = ','):
```
Не использован выданный материал.

mixeme commented on Лабораторная работа №3/Частотный анализ букв в тексте.py:
```diff
@@ -0,0 +1,81 @@
+# TODO  Напишите функцию count_letters
+def count_letters(str):
```
Подумать над решением по проще, используя выданный материал.

mixeme commented on Лабораторная работа №3/Частотный анализ букв в тексте.py:
```diff
@@ -0,0 +1,81 @@
+
+
+# TODO Напишите функцию calculate_frequency
+def calculate_frequency(arr):
```
Подумать над решением по проще, используя выданный материал.

mixeme reviewed (changes requested):
Не использован выданный материал. Недостаточно комментариев.

Feendiers committed:
Update Поиск индекса элемента в списке товаров.py

Добавил комментарии и изменил код для лучшего соответствия выданному материалу

Feendiers committed:
Update Поиск общих участников.py

Добавлены комментарии, а также код изменен под лучшее соответствие выданному материалу

Feendiers committed:
Update Частотный анализ букв в тексте.py

Найдено более простое решение, а также добавлены комментарии

mixeme commented on Лабораторная работа №3/Частотный анализ букв в тексте.py:
```diff
@@ -0,0 +1,63 @@
+                alphabet[alpha] += 1
+            else: alphabet[alpha] = 1#Иначе создаю новое значение словаря с буквой и делаю количество равным 1
+            k += 1 #Tак как символ является буквой, увеличиваю счетчик
+    return alphabet, k#Возвращаю словарь с буквами и количеством каждой буквы по отдельности,
```
Незачем специально отдельно считать количество букв.

mixeme reviewed (changes requested):
Доработать.

Feendiers committed:
Update Частотный анализ букв в тексте.py

Переделан способ подсчета кол-ва букв в тексте

mixeme reviewed (approved):
Принято.
